src/lib/network/wicked.rb

```diff
@@ -64,6 +64,7 @@ def parse_hostname(iface)
     # @param query [String] xpath query. See man wicked for info what is supported there.
     # @return [String] result of the query
     def query_wicked(iface, query)
+      Yast.import "NetworkService"
       raise ArgumentError, "A network device has to be specified" if iface.nil? || iface.empty?
       raise "Parsing not supported for network service in use" if !NetworkService.is_wicked
```

Comment:
why not to use ```config&.backend?(:wicked)``` even here when already massively replacing old calls?

Reply:
I was in doubt about these query methods, because they depend on the service in use and not strictly in the one configured in the config. THat is why I have not modified here.

If you check, there are still some references to NetworkService, specially when them does not depend on the config at all.